submitted_models/marble_husky_sensor_config_3/meshes/VLP16_base_2.dae

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
Can this be deduplicated with VLP base 1?

Reply:
There is a small little knob representing the plugin of the sensor that differs base 1 and base 2 but the design could be simplified and the knobless base could be used for the top and bottom. Let me know if that's what we should do.